src/sentry/grouping/enhancer/enhancement-configs/mobile@2021-04-02.txt

```diff
@@ -88,6 +88,8 @@ family:native function:memcpy category=system
 family:native function:__memcpy category=system
 family:native function:memcmp category=system
 family:native package:CoreFoundation category=system
+family:native package:Foundation category=system
+family:native package:CFNetwork category=system
```

Comment:
Should we just add the full list of [Apple system frameworks](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/OSX_Technology_Overview/SystemFrameworks/SystemFrameworks.html) here?

Reply:
I'll create a separate task for this where we get testcases for each of those symbols and potentially have finer-grained categorization. In principle we should alredy classify most of them via their paths on macos, just not on iOS.